pkg/datapath/loader/template.go

```diff
@@ -28,6 +28,7 @@ const (
 	templateLxcID               = uint16(65535)
 	templatePolicyVerdictFilter = uint32(0xffff)
 	templateIfIndex             = math.MaxUint32
+	templateEndpointNetNsCookie = uint64(0)
```

Comment:
This probably needs to be non-zero to avoid clang inlining things.

Reply:
Done.